### ISSUE TITLE
[Bug]:有一个单测，打开race ，并且设置count 100会跑出来race

### ISSUE THREAD

**Issue by fengttt** [matrixorigin/matrixone#10190]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

No idea, but @XuPeng-SH knows.

**Comment by fengttt** [comment]
Should be fixed by #10188 .    Please resolve when it get merged.

**Comment by XuPeng-SH** [comment]
fixed by #10188 
```
go test -run="TestLogailAppend" -race -timeout=100s --count 100 -failfast github.com/matrixorigin/matrixone/pkg/vm/engine/tae/db/...
```

**Comment by XuPeng-SH** [comment]
@fengttt it is done, pls close this issue
